packages/vscode-extension/src/webview/components/Feedback.tsx

```diff
@@ -0,0 +1,77 @@
+import { Dispatch, MouseEvent, SetStateAction } from "react";
+import "./Feedback.css";
+import classNames from "classnames";
+import { useUtils } from "../providers/UtilsProvider";
+import { Sentiment } from "./SendFeedbackItem";
+
+type FeedbackProps = {
+  sentiment: Sentiment | undefined;
+  setSentiment: Dispatch<SetStateAction<Sentiment | undefined>>;
+};
+
+export function Feedback({ sentiment, setSentiment }: FeedbackProps) {
+  const { sendTelemetry } = useUtils();
```

Comment:
If we have a separate telemetry hook, it sounds like `sendTelemetry` should be there